nncf/common/graph/graph.py

```diff
@@ -162,32 +178,26 @@ class NNCFGraph:
     providing some useful methods for graph traversal.
     """
 
-    ID_NODE_ATTR = "id"
-    KEY_NODE_ATTR = "key"
-    NODE_NAME_ATTR = "node_name"
-    NODE_TYPE_ATTR = "type"
-    METATYPE_ATTR = "metatype"
-    LAYER_NAME_ATTR = "layer_name"
-    LAYER_ATTRIBUTES = "layer_attributes"
     ACTIVATION_SHAPE_EDGE_ATTR = "activation_shape"
     INPUT_PORT_ID_EDGE_ATTR = "input_port_id"
     OUTPUT_PORT_ID_EDGE_ATTR = "output_port_id"
-    IGNORED_ALGOS_ATTR = "ignored_algos"
-    IS_IN_ITERATION_SCOPE_NODE_ATTR = "is_in_iteration_scope"
-    IS_INTEGER_INPUT_NODE_ATTR = "is_integer_input"
     DTYPE_EDGE_ATTR = "dtype"
-    IS_SHARED_ATTR = "is_shared"
     PARALLEL_INPUT_PORT_IDS_ATTR = "parallel_input_ports"
 
     def __init__(self):
         self._nx_graph = nx.DiGraph()
         self._node_id_to_key_dict = {}
+        self._nodes = {}  # type: Dict[str, NNCFNode]
```

Comment:
```suggestion
        self._nodes: Dict[str, NNCFNode] = {} 
```

`type comments` use to support python2. 


`self._nodes` is the main change that improves performance, isn't it?